src/interfaces/ISablierV2ProxyTarget.sol

```diff
@@ -12,7 +12,7 @@ import { Permit2Params } from "../types/Permit2.sol";
 
 /// @title ISablierV2ProxyTarget
 /// @notice Proxy target with stateless scripts for interacting with Sablier V2, designed to be used by
-/// stream senders.
+/// stream senders, recipients and the recipients' approved operators.
```

Comment:
The proxy target is designed to be used exclusively by the stream sender.

Whatever recipient-only functions (e.g., `burn`, ref https://github.com/sablier-labs/v2-periphery/pull/96) have been added only as a contingency - that is, if the proxy target ends up _accidentally_ used by stream recipients.